src/internal/components/live-region/index.tsx

```diff
@@ -76,9 +95,32 @@ function LiveRegion({
     does not impact the performance. If it does, prefer using a string as children prop.
   */
   useEffect(() => {
+    function getSourceContent() {
+      if (source) {
+        return source
+          .map(source => {
```

Comment:
having `source` as the variable name for both the parent and child is a bit hard to keep track of here